evalml/pipelines/components/transformers/preprocessing/time_series_featurizer.py

```diff
@@ -288,6 +288,7 @@ def transform(self, X, y=None):
         delayed_features = self._compute_delays(X_ww, y)
         rolling_means = self._compute_rolling_transforms(X_ww, y, original_features)
         features = ww.concat_columns([delayed_features, rolling_means])
+        features.ww.init()
```

Comment:
Can we reuse any part of the initial schema or use what we know about the dtypes of these features here to reduce the amount of type reinference this might introduce?